polygeist/tools/cgeist/Lib/clang-mlir.h

```diff
@@ -63,18 +63,18 @@ struct LoopContext {
 class BinOpInfo;
 
 /// Context in which a function is located.
```

Comment:
Nit: Should probably update this comment to also mention globals.

Reply:
done.